src/CSharpMongoMigrations.Tests/Migrations/VersionedMigrationTests.cs

```diff
@@ -10,9 +10,9 @@ public class VersionedMigrationFacts
         public void CtsFact()
         {
             // Arrange
-            var migration = new Mock<IMigration>();
+            var migration = new Mock<IMigration>();            
```

Comment:
Rollback such changes please. We don't need to introduce changeset with some extra spaces. 
On the other hand, if you remove extra trailing spaces, it's fine.

Reply:
Done.